core/encoding/json/parser.odin

```diff
@@ -263,8 +263,17 @@ parse_object_body :: proc(p: ^Parser, end_token: Token_Kind) -> (obj: Object, er
 			return
 		}
 
-		obj[key] = elem
-		
+		// NOTE(gonz): There are code paths for which this traversal ends up
+		// inserting empty key/values into the object and for those we do not
+		// want to allocate anything
+		if key != "" {
```

Comment:
I don't particularly like this solution here but as the note says there are code paths than run through here with `key` being the empty string and these are heavily tested for right now.